src/_layouts/atom.liquid

```diff
@@ -0,0 +1,39 @@
+---
+layout: null
+spec: https://tools.ietf.org/html/rfc4287 
```

Comment:
No - it's more of a comment so that people updating the feed can easily refer back to the atom spec.